patterns/hidden-comments.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Title: Comments
+ * Slug: twentytwentyfive/hidden-comments
+ * Inserter: no
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:comments {"className":"wp-block-comments-query-loop","style":{"spacing":{"margin":{"top":"var:preset|spacing|70","bottom":"var:preset|spacing|70"}}}} -->
+<div class="wp-block-comments wp-block-comments-query-loop" style="margin-top:var(--wp--preset--spacing--70);margin-bottom:var(--wp--preset--spacing--70)">
+	<!-- wp:heading -->
+	<h4><?php esc_html_e( 'Comments', 'twentytwentyfive' ); ?></h4>
```

Comment:
This heading is missing the block class name and is creating a block validation error.

Reply:
Why is the H4 heading used twice? The hierarchy is incorrect.